Getting_Started.md

```diff
@@ -0,0 +1,128 @@
+    
+    [farms]
+    ossbunsen-farm-f36-1
+    
+    [all:vars]
+    deploying_account = admin
+    target_account = admin
+    ```
+
+    The `admin` account should be replaced with the account that you intend to do your development with.
+
+    NOTE: In place of `ansible_ssh_pass=vagrant`, you can use `ansible_ssh_private_key_file='/path/to/vagrantdir/.vagrant/machines/<name>/libvirt/private_key'` (of course if you're not using vagrant or you have another private key granted, then you should point at that instead)
+        - `/path/to/vagrantdir` would be where you cloned [the VDO vagrant example repository](https://github.com/dm-vdo/vagrant)
+
+    NOTE: You can also leave out `ansible_ssh_pass` and `ansible_ssh_private_key_file` and insted use `--ask-pass` when running the `ansible-playbook` commands below to be prompted for the passwords instead.
+
+3. Now we can run the provisioning for the first time.  It is generally recommended to provision the roles serially to ensure that any dependencies across roles are satisfied without the chance of a timing issue being introduced.
+
+    Example commands for each role.
+    ```
+    ansible-playbook -i inventory \
+                     -l infrastructure \
+                     provisioning/playbook.yml \
+                     --extra-vars="install_bunsen_support=true" \
+                     --ask-become-pass
+
+    ansible-playbook -i inventory \
+                     -l resources \
+                     provisioning/playbook.yml
+
+    ansible-playbook -i inventory \
+                     -l farms \
+                     provisioning/playbook.yml
+    ```
+
+    - Note the added options `--extra-vars="install_bunsen_support=true" --ask-become-pass` on the infrastructure role.
+        - `--extra-vars="install_bunsen_support=true"` instructs the ansible playbook to install the necessary COPR repositories on the `controller` system and install the python dependencies (python3-resource-discovery, and python3-utility-mill).  It is only needed to be specified one time.
+        - `--ask-become-pass` instructs the `ansible-playbook` command to prompr for the sudoers password on the `controller` to allow for you to install the packages.
+        - If you would prefer NOT letting ansible provision your `controller`, please see the [Manually Installing Bunsen Support Libs](#installing-bunsen-support-libraries)
+      
+# Using the Bunsen environment
+
+While the environment is yours and you can make it work as you wish, the most effective workflow involves doing general development on the `resource` machine and then doing the compilations and testing from that machine.
+
+The home directory for the `target_account` specified above is mounted via NFS, so if you are doing work that requires multiple distributions, you don't have to unnecessarily duplicate things.
+
+To perform general development and testing for VDO, please see the VDO Getting Started document (This does not exist yet, contents TBD)
```

Comment:
Line 103: Is this referring to this Getting_started.md, or something else? It's a bit confusing.

Reply:
Something else that does not exist yet.  That's what I was referring to in my comment above as hoping to write this week.

Reply:
This will be addressed at a later time (maybe tonight?) :)